satellite/satellitedb/irreparable_test.go

```diff
@@ -1,28 +1,17 @@
 // Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package irreparabledb_test
+package satellitedb
```

Comment:
I think this test file should be placed next to `IrreparableDB` interface in `datarepair` package.

Reply:
done